authorizer-app/src/app/components/rest-source-authorization/update-rest-source-user.component.html

```diff
@@ -9,11 +9,11 @@
 
 <div *ngIf="restSourceUser" class="container col-md-4">
   <h3 *ngIf="!isEditing">Enter details for User</h3>
-  <h3 *ngIf="isEditing">Edit details for User</h3>
+  <h3 *ngIf="isEditing">Details for User</h3>
   <div class="form-group">
     <label for="externalUserId">External User ID:</label>
```

Comment:
Perhaps rename to (REST?) Service user ID

Reply:
we used `external-user-id` or `External User ID` in many places. You mean change them to `service-user-id` or `Service User ID`?

Reply:
I think we don't need the serviceUserId in the table, as long as users can see it in the info dialog.